akka-docs/src/main/paradox/typed/persistence.md

```diff
@@ -666,6 +667,16 @@ active entity (`EventSourcedBehavior`) for each id within a data center.
 @ref:[Replicated Event Sourcing](replicated-eventsourcing.md) supports active-active persistent entities across
 data centers.
 
+## Internal logging
+
+Behind the scenes, a `EventSourcedBehavior` performs different operations like loading snapshots, handling commands, applying and replaying events.
+These operations are logged in debug level using a dedicated internal logger that can be enabled when desired. 
+
+The logger is named after the `ActorContext` logger name, but with `Internal` appended to it. 
+For instance, if the `ActorContext` logger name is `application.TaskManager`, the internal logger will be named `application.TaskManagerInternal`.
```

Comment:
shouldn't it be `application.TaskManager.Internal`? logback configuration is often by such prefix with `.` notation. Maybe it's the same without `.` but I'm thinking it would be more conventional to use `.`

Reply:
I choose not to add a `.` because if defining the bevahior inside a scala object, you get `application.TaskManager$` and the internal would be `application.TaskManager$Internal`. I wanted to avoid `application.TaskManager$.Internal`. 

Anyway, I will remove this and move to `akka.persistence.typed.internal.EventSourcedBehaviorImpl`. I think it will be too confusing for users to have an Internal one based on their own class name.